src/AbstractExtendedPdo.php

```diff
@@ -564,18 +566,18 @@ public function perform(string $statement, array $values = []): PDOStatement
      *
      * @param string $query The SQL statement to prepare for execution.
      *
-     * @param array $options Set these attributes on the returned
+     * @param array|null $options Set these attributes on the returned
      * PDOStatement.
      *
      * @return PDOStatement
      *
      * @see http://php.net/manual/en/pdo.prepare.php
      *
      */
-    public function prepare(string $query, array $options = []): PDOStatement
+    public function prepare($query, $options = null): PDOStatement
```

Comment:
3.x
https://github.com/auraphp/Aura.Sql/blob/f19b1aa810b35d86ad55562876d967ef3f9a97b7/src/AbstractExtendedPdo.php#L577

The `$options` default value is `[]` on 3.x.
Why do you need to change to `null`?

Reply:
I think this was a signature issue with PDO, but I'll double check.

Reply:
It seems `$options` default value is `[]`.
>  public PDO::prepare(string $query, array $options = []): PDOStatement|false
https://www.php.net/manual/en/pdo.prepare.php#refsect1-pdo.prepare-description

Can you please explain it should be null?

Reply:
Sorry, thought I had posted the comment above. Hopefully that clears things up.

Reply:
Thank you!